src/main/java/com/swervedrivespecialties/exampleswerve/util/BeakXboxController.java

```diff
@@ -1,13 +1,50 @@
 package com.swervedrivespecialties.autonrefresh.util;
 
-import com.swervedrivespecialties.autonrefresh.RobotMap;
-
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import edu.wpi.first.wpilibj.GenericHID.Hand;
+import edu.wpi.first.wpilibj2.command.button.Button;
 
 public class BeakXboxController{
 
-    private XboxController controller;
+    //these constants all come from the layout of our physical controllers
+    //and their IO interfaces with WPILib
+
+    //deadband
+    private static final double DEADBAND = .05;
+
+    //button indexing
+    private static final int BUTTON_A_INDEX = 1;
+    private static final int BUTTON_B_INDEX = 2;
+    private static final int BUTTON_X_INDEX = 3;
+    private static final int BUTTON_Y_INDEX = 4;
+    private static final int BUTTON_LB_INDEX = 5;
+    private static final int BUTTON_RB_INDEX = 6;
+    private static final int BUTTON_BACK_INDEX = 7;
+    private static final int BUTTON_START_INDEX = 8;
+
+    //trigger identification booleans
+    private static final boolean IS_LEFT_TRIGGER_DEFAULT = true;
+    private static final boolean LEFT_TRIGGER_BOOLEAN = IS_LEFT_TRIGGER_DEFAULT;
+    private static final boolean RIGHT_TRIGGER_BOOLEAN = !IS_LEFT_TRIGGER_DEFAULT;
+
+    //axis indexing
+    private static final int LEFT_X_AXIS_INDEX = 0;
+    private static final int LEFT_Y_AXIS_INDEX = 1;
+    private static final int LEFT_TRIGGER_AXIS_INDEX = 2;
+    private static final int RIGHT_TRIGGER_AXIS_INDEX = 3;
+    private static final int RIGHT_X_AXIS_INDEX = 4;
+    private static final int RIGHT_Y_AXIS_INDEX = 5;
+
+    //DPAD degree values; like a clock--0 at top, clockwise is positive
+    private static final int DPAD_UP_DEGREES = 0;
```

Comment:
Same goes for these, they seem more with the style you were going for. Just a little more consistent with it.